applications/workers/workers-smoketest/src/smokeTest/kotlin/net/corda/applications/workers/smoketest/ClusterBootstrapTest.kt

```diff
@@ -35,7 +35,7 @@ class ClusterBootstrapTest {
     )
     private val client = HttpClient.newBuilder().build()
 
-    @Test
+    //@Test
```

Comment:
We probably should restore:
```suggestion
    @Test
```

Reply:
fixed